lib/views/pay/pay_view.dart

```diff
@@ -101,7 +101,7 @@ class _PayViewState extends State<PayView> {
           paymentResponse != null
               ? paymentResponse!.payResponse["Error"] == null
                   ? Text(
-                      "Payment Successfully : ${paymentResponse!.payResponse["amountMsat"]["msat"]} msats")
+                      "Payment Successfully done! : ${paymentResponse!.payResponse["amount_msat"]}")
```

Comment:
No this do not work too.

Look how it is done this model https://github.com/dart-lightning/lndart.clnapp/blob/main/lib/model/app_model/list_funds.dart and how it is used inside the code

Reply:
Yes, I figured that out,